src/forms/roomSchema.ts

```diff
@@ -0,0 +1,30 @@
+import * as Yup from "yup";
+
+import { ROOM_TAXON } from "settings";
+
+import { createNameSchema } from "forms/factory/createNameSchema";
+
+import { createUrlSchema } from "./factory/createUrlSchema";
+
+export interface RoomSchemaShape {
+  title: string;
+  venueName?: string;
+  url?: string;
+  useUrl?: boolean;
+  image_url: string;
+}
+
+const roomImageUrlSchema = Yup.string().required(
+  `${ROOM_TAXON.capital} icon is required`
+);
```

Comment:
Shouldn't this be inlined with the rest of the fields? Not really that complicated case to warrant its own name.

Reply:
done 3b82563